src/selectors/forms.js

```diff
@@ -43,21 +42,3 @@ export const makeRehydrateForm = () =>
     [propStageForm, protocolForms],
     (form, forms) => forms[form] || null,
   );
-
-/**
- * @return {Object} keyed by form name
- */
-export const getDefaultFormValues = createSelector(
```

Comment:
This seems to be needed to populate hidden fields on the NodeForm. To test:

1. Start session with dev protocol
2. Fill out & submit form to create new node

Variable "04298634..." should equal 'network-canvas', but is null in this branch.

Reply:
So _that's_ what this was for! Thanks.

There's no mechanism currently to create hidden variables in a form in Architect or elsewhere. I feel like I've been back and forth on this a million times, but for the sake of clarity let's just remove hidden inputs types.

Reply:
Sounds good; I imagine `additionalProperties` covers most of the same use cases. I opened #843 to cover the rest of this.

Reply:
Thanks for the open issue.